### PR TITLE
Remove Grok Dependency for Vocabularies.

### PR DESCRIPTION
**Pull request by l34marr**
Hopefully this is a step toward removing Grok dependency.

refs. #510

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -32,6 +32,8 @@ There's a frood who really knows where his towel is.
 - Clean up static files.
   [rodfersou]
 
+- Remove Grok dependency for vocabularies.
```
Comment (hvelarde) [inline CHANGES.rst]: new chagelog entries on top of the list, please

File: src/collective/cover/vocabularies.py [hunk 2]
```diff
@@ -3,20 +3,20 @@
...
 from zope.component import queryUtility
+from zope.interface import implements
 from zope.schema.interfaces import IVocabularyFactory
 from zope.schema.vocabulary import SimpleTerm
 from zope.schema.vocabulary import SimpleVocabulary
 
 
 class AvailableLayoutsVocabulary(object):
-    grok.implements(IVocabularyFactory)
+    implements(IVocabularyFactory)
```
Comment (hvelarde) [inline src/collective/cover/vocabularies.py]: would you mind use the `@implementer` decorator instead?

**Comment by hvelarde** [comment]
thank you for taking care of this!

**Comment by l34marr** [comment]
For the record: ref to http://github.com/eea/eea.faceted.vocabularies/blob/master/eea/faceted/vocabularies/configure.zcml and http://github.com/plone/plone.app.multilingual/blob/master/src/plone/app/multilingual/browser/configure.zcml

Hopefully this will not break anything.

**Pull request merged**